scripts/koji/koji_build.py

```diff
@@ -1,37 +1,47 @@
 #!/usr/bin/env python3
 import argparse
+import logging
 import os
 import subprocess
 import re
+from contextlib import contextmanager
+from datetime import datetime, timedelta
+from pathlib import Path
+
+from specfile import Specfile
```

Comment:
I was testing but forgot to add the extra dependency. And this will happen to others, as we don't package it at the moment.

```
~/Vates/git/rpms/dummypkg [stormi-test L ✔] $ koji_build.py --help
Traceback (most recent call last):
  File "/home/samuel/Vates/git/xcp/scripts/koji/koji_build.py", line 11, in <module>
    from specfile import Specfile
ModuleNotFoundError: No module named 'specfile'
```

I suggest catching the error and turning it into a nice message on the console, for a smoother transition of existing users of this script.